sql-cli/sql_cli/__main__.py

```diff
@@ -51,5 +51,59 @@ def validate(environment: str = "default", connection: Optional[str] = None) ->
     validate_connections(environment=environment, connection_id=connection)
 
 
+@app.command()
+def init(
+    project_dir: Optional[Path] = typer.Argument(
+        None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
+    ),
+    airflow_home: Optional[Path] = typer.Option(
+        None,
+        dir_okay=True,
+        help=f"(Optional) Set the Airflow Home. Default: {configuration.DEFAULT_AIRFLOW_HOME}",
+        show_default=False,
+    ),
+    airflow_dags_folder: Optional[Path] = typer.Option(
+        None,
+        dir_okay=True,
+        help=f"(Optional) Set the DAGs Folder. Default: {configuration.DEFAULT_DAGS_FOLDER}",
+        show_default=False,
+    ),
+) -> None:
+    """
+    Initialise a project structure to write workflows using SQL files.
+
+    \b\n
+    Examples of usage:
+    \b\n
+    $ flow init
+    \b\n
+    $ flow init .
+    \b\n
+    $ flow init project_name
+
+
+    \b\n
+    By default, the project structure includes:
+
+    ├── config: withholds configuration, e.g. database connections, within each environment directory
+    \b\n
+    ├── data: directory which contains datasets, including SQLite databases used by the examples
+    \b\n
+    └── workflows: directory where SQL workflows are declared, by default has two examples of workflow
+
+    \b\n
+    Next steps:
+    \b\n
+    * Update the file `config/default/configuration.yaml` to declare database connections.
+    \b\n
+    * Create SQL workflows within the `workflows` folder.
+    """
+    project_dir = project_dir or Path(os.getcwd())
+
+    proj = project.Project(project_dir)
+    proj.initialise(airflow_home, airflow_dags_folder)
```

Comment:
I like that more. 👍 I realised after talking to you about that, that the project init with creating dirs, and so on is to much for "hiding" it / doing it in the init.

Reply:
Agreed, @feluelle , we can have a method to check if the given directory is a SQL CLI project - I'll be doing this in the next PR, when I align a bit more the validate and init!